cyber/common/global_data.cc

```diff
@@ -102,7 +102,7 @@ bool GlobalData::IsRealityMode() const { return is_reality_mode_; }
 
 void GlobalData::InitHostInfo() {
   char host_name[1024];
-  gethostname(host_name, 1024);
+  gethostname(host_name, sizeof(host_name));
```

Comment:
@freeHackOfJeff : and this way you are asking for something already known in the same scope? Why don't we create const member to cover line 104 and 105?

Reply:
> @freeHackOfJeff : and this way you are asking for something already known in the same scope? Why don't we create const member to cover line 104 and 105?

Thanks for your comments. I keep it as current implementation because it looks simple in this case. Hence, define another const member looks a bit redundant to me. from another point of view, the maximum length of hostname is defined and limited by POSIX_HOST_NAME_MAX and MAXHOSTNAMELEN to 256 with the tail character as zero in Glibc. Also, consider the architecture, I have an idea to encapsulate network related stuff in a network file if you look through Apollo network related code in other modules, but no time currently. That's would be more than welcome if anybody would like to work on that if sounds make sense. :)